app/src/components/widgets/multiLevelWidgets/componentHealthCheckTable/columns/nameColumn.jsx

```diff
@@ -23,26 +25,56 @@ import styles from '../componentHealthCheckTable.scss';
 
 const cx = classNames.bind(styles);
 
+const NameColumnAttribute = ({ value, onClickAttribute, isClickableAttribute, minPassingRate }) => {
+  const color = value.passingRate < minPassingRate ? COLOR_DEEP_RED : COLOR_PASSED;
+
+  return (
+    <div
+      className={cx('name-attr', { 'cursor-pointer': isClickableAttribute })}
+      onClick={
+        isClickableAttribute
+          ? () => onClickAttribute(value.attributeValue, value.passingRate, color)
+          : undefined
+      }
+    >
+      <span>{value.attributeValue}</span>
+    </div>
+  );
+};
+NameColumnAttribute.propTypes = {
+  value: PropTypes.object,
+  onClickAttribute: PropTypes.func,
+  isClickableAttribute: PropTypes.bool,
+  minPassingRate: PropTypes.number,
+};
+NameColumnAttribute.defaultProps = {
+  value: {},
+  onClickAttribute: () => {},
+  isClickableAttribute: false,
+  minPassingRate: 0,
+};
+
+const NameColumnAttributeWithTooltip = withTooltip({
+  TooltipComponent: TextTooltip,
+  data: {
+    dynamicWidth: true,
+    placement: 'right',
+    tooltipTriggerClass: cx('tooltip-trigger'),
```

Comment:
I don't see this class name in `componentHealthCheckTable.scss`. Is it just copied from somewhere?